_posts/2024-08-12-AI-chatting.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Introducing AI features to JabRef"
+tags: [AI, GSoC]
+author: "[Ruslan](https://github.com/InAnYan)"
+---
+
+Hello, everyone! My name is Ruslan and I'm a novice JabRef contributor and I'm working on AI project for [Google Summer of Code](https://summerofcode.withgoogle.com/).
+
+I want to introduce you to the new AI features in JabRef.
+
+- AI can generate for you a summary of a research paper
+- You can also chat with papers using a smart AI assistant
+
+## AI summary tab
+
+We have made a new entry editor tab: "AI Summary", where AI will generate for you a quick overview of the paper.
+
+![AI summary tab screenshot](../img/AiSummary.png)
+
+The AI will mention for you main objectives of the research, methods used, key findings, and conclusions.
+
+## AI chat tab
+
+The next new entry editor tab is "AI chat", where all the question and answering (Q&A) happens.
+
+![AI chat tab screenshot](../img/AiChat.png)
+
+In this window you can see the following elements:
+
+- Chat history with your messages
+- Prompt for sending messages
+- A button for clearing the chat history (just in case)
+
+Let's try it out on a paper "Cooper, K., Donovan, J., Waterhouse, A., & Williamson, G. (2007). Cocoa and health: a decade of research. *British Journal of Nutrition*, 99(1), 1–11. doi:[10.1017/s0007114507795296](https://doi.org/10.1017/s0007114507795296)"
+
+Let's ask about the chocolate.
+
+![AI first question and answer](../img/AiQuestion1.png)
+
+Correct! The authors talk a lot about chocolate components, and how it can be a powerfull antioxidant.
+Wow! AI has given so much infromation, and it's true: chocolate really contains polyphenols and fatty acids.
+
+I was wondering, how much chocolate should a human eat in a day? Let's dive in this topic.
+
+![AI second question and answer](../img/AiQuestion2.png)
+
+*Only 40 grams?* Well, our health is our wealth. And AI helped us to uncover this information!
+
+## How does this work?
+
+In the background, JabRef analyses the linked PDF files of library entries. The information used after the indexing is then supplied to the AI, which, to be precise, in our case is a Large Language Model (LLM).
+The LLM is currently not stored on your computer. Instead, we have many integrations with AI providers (OpenAI, Mistral AI, Hugging Face), so you can choose the one you like the most.
```

Comment:
You had links to all providers. Can you add them here?

Reply:
No, not here. If you want them listed, then better in the section down below --> "Which AI provider should I choose". This section here is about "how does this work".

Reply:
My reviewing "driver": Science: Link at first appearance. 

Maybe the provide list here can be removed. Does it add value to have the concrete list here?